tests/run_tests.py

```diff
@@ -43,14 +43,20 @@
     "multiple_ubo_update_with_graphics_pipeline.vkscript",
     # DXC not currently building on bot
     "draw_triangle_list_hlsl.amber",
+    # Texture file not found from this path
```

Comment:
If this is suppressed on all platforms, where does this work? Why is the path not found?

Reply:
I'm considering it more as a sample than a test since it doesn't have any EXPECT commands. The texture file is under tests/cases, but the script is run under tests/ so it doesn't find the file. I could change the file name in the amber file, but then it wouldn't work when running manually from the same path as the amber file.

Reply:
Right, I think we need to setup everything so the file path is relative to the `.amber` file. This means that `amber.cc` will need to store the path to the `.amber` and prepend that to the filename before calling the loader.

Reply:
Ok, sounds reasonable.